docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.6'
+
+services:
+  dmarc-report:
+    image: dmarc-report:latest
+    hostname: dmarc-report
+    container_name: dmarc-report
+    restart: always
+    depends_on:
+      - db
+    ports:
+      - "4446:80"
```

Comment:
can we make this at least 8080:80 or something like this? I don't think a port like 4446 adds to the example clarity here, even if this is maybe the port you are using, but it should denote that it is a http service (hence I'm suggesting 8080 as alt-http)

Reply:
sure